sip/execution_framework_interface/docker_api/README.md

```diff
@@ -0,0 +1,41 @@
+# Docker Client API
+
+## Introduction
+
+The SIP Execution Framework provides a Docker Client Python API to run
+docker services by various Execution Control Service components.
+
+The client library is written as a of Python modules which are structured as a 
+set of higher level modules use by various Execution Control Services. 
+The client lets you run docker commands from within Python - create services, 
+delete services, manage nodes etc
+
+## Quick-start
+
+To use the client, simply copy the client script to the service directory.
+
+import the client and initialise the client.
+
+```bash
+from ..docker_client import DockerClient
+
+DC = DockerClient()
+```
+
+## Testing
```

Comment:
Should probably add a warning this section to say that the unit test will do potentially nasty things like delete all docker volumes. Could give someone a nasty surprise if they don't know first!